kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -187,6 +183,8 @@ usageRightsEditor.controller(
     function uiError(error) {
         ctrl.error = error.body.errorMessage;
     }
+
+    init();
```

Comment:
> We could either memoize the categories in the service

The service already does this - https://github.com/guardian/grid/blob/master/kahuna/public/js/services/api/edits-api.js#L13